### PR TITLE
Update to docker-compose v3

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py [hunk 1]
```diff
@@ -72,6 +73,8 @@
 
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+if not config('DATABASE_URL').startswith('sqlite') and not wait_for_url(config('DATABASE_URL')):
```
Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: @pmac is this the way you intended to use `urlwait`? Also is there another way to work around `sqlite` urls?

Comment (pmac) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: I didn't think of doing it in settings... could work. I mostly conceived of using it as part of a script like `run-dev.sh`. You'd just put a call to `urlwait` before the command to run the site and it'd do the right thing.

Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: Hmm I like having the sqlite option, i guess if could do the check in the bash script too.

Comment (pmac) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: Maybe I should add a check in the lib itself. Seems appropriate.

Comment (pmclanahan) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: @glogiotatidis urlwait 0.4 has been released and is on pypi.

Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: 👍 thanks!

Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py]: Moved `urlwait` to `run-dev.sh`

**Comment by glogiotatidis** [comment]
Update to urlwait=4 and moved `urlwait` to run-dev as suggested. Good to go @pmclanahan ?

**Comment by glogiotatidis** [comment]
Thanks!

File: {{ cookiecutter.project_name }}/bin/run-dev.sh [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
+urlwait  # Wait for the database to come online
```
Comment (peterbe) [inline {{ cookiecutter.project_name }}/bin/run-dev.sh]: How does it know which URL to wait for?

Comment (pmac) [inline {{ cookiecutter.project_name }}/bin/run-dev.sh]: It defaults to `DATABASE_URL`, but you can [use the CLI](https://github.com/pmclanahan/urlwait#usage) to specify a URL or a different variable to watch.

**Pull request merged**